src/data.js

```diff
@@ -0,0 +1,35 @@
+const Pieces = [
```

Comment:
You shouldn't have Piece stuff in here. You can bring in the FILES and RANKS in there though, and this can be used inside board.js as well as the various tests - a good reuse of code constants.